android/modules/ui/src/java/ti/modules/titanium/ui/widget/tabgroup/TiUIAbstractTabGroup.java

```diff
@@ -169,22 +172,31 @@ public TiUIAbstractTabGroup(final TabGroupProxy proxy, TiBaseActivity activity)
 	{
 		super(proxy);
 
+		// Determines if theme has the "titaniumIsSolidTheme" attribute and it's set to "true".
+		// Used by our "Theme.Titanium.*.Solid" themes to shade the top/bottom tabs appropriately.
+		TypedValue typedValue = new TypedValue();
+		activity.getTheme().resolveAttribute(R.attr.titaniumIsSolidTheme, typedValue, true);
```

Comment:
Maybe this check should be a method in _TiApplication.java_?

Reply:
I'll make a method in [TiUIHelper.java](https://github.com/appcelerator/titanium_mobile/blob/master/android/titanium/src/java/org/appcelerator/titanium/util/TiUIHelper.java) since it already has an `isUsingMaterialTheme()` method.